pkgs/build-support/docker/default.nix

```diff
@@ -935,33 +930,39 @@ rec {
           # following lines, double-check that your code behaves properly
           # when the number of layers equals:
           #      maxLayers-1, maxLayers, and maxLayers+1, 0
-          store_layers="$(
-            paths |
-              jq -sR '
-                rtrimstr("\n") | split("\n")
-                  | (.[:$maxLayers-1] | map([.])) + [ .[$maxLayers-1:] ]
-                  | map(select(length > 0))
-              ' \
-                --argjson maxLayers "$availableLayers"
-          )"
+          paths |
+            jq -sR '
+              rtrimstr("\n") | split("\n")
+                | (.[:$maxLayers-1] | map([.])) + [ .[$maxLayers-1:] ]
+                | map(select(length > 0))
+            ' \
+              --argjson maxLayers "$availableLayers" > store_layers.json
```

Comment:
I'm sorry, it's unclear to me from the way GitHub displays this diff what exactly it is you're suggesting (except perhaps changing a random space into a \t but I doubt it). However it seems it can be directly applied by GitHub so feel free.

Reply:
It's just the two spaces I think.

> can be directly applied by GitHub

Yet, it doesn't give me the button to apply it. Let me try another (same) suggestion:

```suggestion
            jq -sR '
              rtrimstr("\n") | split("\n")
                | (.[:$maxLayers-1] | map([.])) + [ .[$maxLayers-1:] ]
                | map(select(length > 0))
              ' \
              --argjson maxLayers "$availableLayers" > store_layers.json
```

Reply:
oooooh, now I see! I was distracted by all the red "highlights" that weren't changes...

Because you wanted to squash the commits I've updated the branch manually rather than through GitHub's suggestion.